patches/tModLoader/Terraria/Main.cs.patch

```diff
@@ -1024,7 +1024,7 @@
  				Console.WriteLine(textValue);
  				Console.WriteLine(textValue2);
 +				Console.WriteLine("m\t\tMods Menu");
-+				Console.WriteLine("b\t\tMod Browser");
++				//Console.WriteLine("b\t\tMod Browser"); // Not functional in the 1.4 Steam Workshop Environment 
```

Comment:
Literally can't run without a Steam Desktop Client being installed, which means it can't run on a headless server, making it effectively useless as is in a Steam Workshop backend.